packages/builders/scripts/inter-protocol/replace-feeDistributor-combo.js

```diff
@@ -0,0 +1,80 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForReplaceFeeDistributor } from '@agoric/inter-protocol/src/proposals/replace-fee-distributor.js';
+import { SECONDS_PER_HOUR } from '@agoric/inter-protocol/src/proposals/econ-behaviors.js';
+
+const configurations = {
+  A3P_INTEGRATION: {
+    params: {
+      collectionInterval: 30n,
+      keywordShares: {
+        RewardDistributor: 0n,
+        Reserve: 1n,
+      },
+    },
+  },
+  MAINNET: {
+    params: {
+      collectionInterval: 1n * SECONDS_PER_HOUR,
+      keywordShares: {
+        RewardDistributor: 0n,
+        Reserve: 1n,
+      },
+    },
+  },
+  // TODO(CTH): fill in something for DEVNET and EMERYNET
```

Comment:
TODO when?

Reply:
If this PR introduce a new proposal builder exercised in the upcoming upgrade, it should contain all config for all networks the upgrade could be deployed to.

Reply:
`upgrade.go` has these four.

Reply:
I see configs for devnet and emerynet now, so satisfied.